### ISSUE TITLE
clear not used / inuse class not removed when custom_func 'Select Value' (remove filter) selected

### ISSUE THREAD

**Issue by xmikew** [vedmack/yadcf#279]
https://jsfiddle.net/t372e7rh/1/ (column 2 is our column of interest)

using a custom_func with data parameter. I noticed that on 0.8.8 when selecting the "Select Value" option did nothing. In 0.8.9b, the values are reset, but the 'inuse' class is never removed. I set a breakpoint on:

In doFilterCustomDateFunc (starting around line 745)
```
		if (arg === "clear") {
			if (exGetColumnFilterVal(oTable, column_number) === '') {
				return;
			}
			$("#yadcf-filter-" + table_selector_jq_friendly + "-" + column_number).val('-1').focus();
			$("#yadcf-filter-" + table_selector_jq_friendly + "-" + column_number).removeClass("inuse");
```

However, the arg is never set to 'clear' -- perhaps this is expected when i do not use a filter_reset_button (set _text to false).

However, inuse class is never removed. when selecting "Select Value" -- I'll dig into a little more tonight but perhaps it would be easier for you to find.

Thanks as always.

\- Mike

**Comment by xmikew** [comment]
using fixed version: https://jsfiddle.net/t372e7rh/2

**Comment by vedmack** [comment]
Final fix in [0.8.9.beta.38](https://github.com/vedmack/yadcf/blob/4b778d7747c5c1cd4f362e6a6154c172f4a88ad3/jquery.dataTables.yadcf.js)

Thanks for your [PR](https://github.com/vedmack/yadcf/pull/280)
